### PR TITLE
add build linux arm64

### PR DESCRIPTION
**Pull request by zhouqiang-cl**

### CODE REVIEW

File: jenkins/pipelines/build-linux-amd64-enterprise.groovy [hunk 1]
```diff
@@ -66,7 +66,7 @@ try {
                     }
                 }
                 dir("go/src/github.com/pingcap/enterprise-plugin") {
-                    checkout changelog: false, poll: true, scm: [$class: 'GitSCM', branches: [[name: "release-4.0"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PruneStaleBranch'], [$class: 'CleanBeforeCheckout'], [$class: 'CloneOption', timeout: 2]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', refspec: '+refs/heads/*:refs/remotes/origin/*', url: 'git@github.com:pingcap/enterprise-plugin.git']]]
+                    checkout changelog: false, poll: true, scm: [$class: 'GitSCM', branches: [[name: "release-5.0"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'PruneStaleBranch'], [$class: 'CleanBeforeCheckout'], [$class: 'CloneOption', timeout: 2]], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', refspec: '+refs/heads/*:refs/remotes/origin/*', url: 'git@github.com:pingcap/enterprise-plugin.git']]]
```
Comment (BusyJay) [inline jenkins/pipelines/build-linux-amd64-enterprise.groovy:69]: Why change it directly instead of creating a new one?

**Review by BusyJay — COMMENTED** [review]
script part for TiKV LGTM

File: jenkins/pipelines/build-linux-amd64-enterprise.groovy (hunk 1 above)
Comment (zhouqiang-cl) [reply jenkins/pipelines/build-linux-amd64-enterprise.groovy:69]: will refine later,

Comment (zhouqiang-cl) [reply jenkins/pipelines/build-linux-amd64-enterprise.groovy:69]: https://github.com/PingCAP-QE/ci/issues/116

**Pull request merged**